spec/models/topic_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+	scenario 'can a topic be saved' do
+      topic = Topic.new(body: "What is your secret superpower?")
```

Comment:
This is what you use FactoryBot for.
```
topic = FactoryBot.build(:topic, body: "What is your...")
```

It makes instantiating objects easier (though there isn't really much advantage here because your model, with only one attribute, is so simple).

(You can also use `FactoryBot::create` to instantiate and save a model to the database)